src/BSeries.jl

```diff
@@ -1800,7 +1842,7 @@ This code is based on the Theorem 2 of
   Foundations of Computational Mathematics 10 (2010): 673-693.
   [DOI: 10.1007/s10208-010-9073-1](https://link.springer.com/article/10.1007/s10208-010-9073-1)
 """
-function is_energy_preserving(series_integrator)
+function is_energy_preserving(series_integrator, tolerance)
```

Comment:
1. The change needs to be reflected in the docstring
2. I would again prefer optional keyword arguments with sensible default values